adoc/admin-gpus.adoc

```diff
@@ -0,0 +1,180 @@
+[[gpus]]
+= NVIDIA GPUs
+
+SUSE's CaaS Platform supports scheduling GPU-dependent workloads on NVIDIA GPUs as a technical preview. This can be a powerful way to run compute-intensive workloads such as machine learning pipelines. This section illustrates how to prepare your host machine to expose GPU devices to your containers, and how to configure Kubernetes to schedule GPU-dependent workloads.
+
+== Prepare the host machine
+
+=== Install the GPU drivers
+
+Not every worker node in the cluster need have a GPU device present. On the nodes that do have one or more NVIDIA GPUs, install the drivers from NVIDIA's repository.
+
+[source,bash]
+----
+# zypper addrepo --refresh https://download.nvidia.com/suse/sle15sp2/ nvidia
+# zypper refresh
+# zypper install x11-video-nvidiaG05
+----
+
+[NOTE]
+====
+For most modern NVIDIA GPUs, the G05 driver will support your device. Some older devices may require the x11-video-nvidiaG04 driver package instead. Check NVIDIA's documentation for your GPU device model.
+====
+
```

Comment:
In my opinion we should put `[warning]` here that this is in tech preview and what that means.

Reply:
Done